buildroot-external/kernel/device-support.config

```diff
@@ -67,7 +67,7 @@ CONFIG_RT2800USB_UNKNOWN=y
 # Multimedia core support
 CONFIG_MEDIA_SUPPORT=y
 CONFIG_MEDIA_CONTROLLER=y
-CONFIG_MEDIA_CONTROLLER_DVB=m
+CONFIG_MEDIA_CONTROLLER_DVB=y
```

Comment:
Where does that y came from? Isn't this global config applied at the end? 🤔

Reply:
It's a `bool`, so it can't have `m`: https://github.com/torvalds/linux/blob/master/drivers/media/mc/Kconfig#L8

AFAIK Kconfig doesn't apply parent's (i.e. what the option `depends` on) value to children, so there is no semantics in setting it to `m`. And since the diff of all the configs before and after these changes is empty, it confirms that.